app/assets/javascripts/discourse/app/lib/plugin-api.js

```diff
@@ -1680,6 +1681,74 @@ class PluginApi {
   addTopicsSectionLink(arg) {
     addSectionLink(arg);
   }
+
+  /**
+   * EXPERIMENTAL. Do not use.
+   * Support for adding a Sidebar section by returning a class which extends from the BaseSectionHeader
```

Comment:
If we're adding a custom Sidebar section, `BaseSectionHeader` doesn't sound right. What do you think about `BaseCustomSidebarSection` instead?